checks/scoring.py

```diff
@@ -235,6 +235,11 @@
 MAIL_AUTH_SPF_ERROR = NO_POINTS
 MAIL_AUTH_SPF_WORST_STATUS = STATUS_FAIL
 
+MAIL_AUTH_TLSRPT_PASS = NO_POINTS
+MAIL_AUTH_TLSRPT_FAIL = NO_POINTS  # TLS-RPT fail does not give a points penalty
+MAIL_AUTH_TLSRPT_ERROR = NO_POINTS
+MAIL_AUTH_TLSRPT_WORST_STATUS = STATUS_FAIL
```

Comment:
Per the [scoring documentation](https://github.com/internetstandards/Internet.nl/blob/main/documentation/scoring.md) this should probably also use FULL/NO/NO_POINTS for PASS/FAIL/ERROR, with the WORST_STATUS as STATUS_INFO (or STATUS_NOTICE if we want to give it more weight). That setting for WORST_STATUS should give it no score impact.